tls/src/main/java/org/bouncycastle/tls/TokenBindingExtension.java

```diff
@@ -0,0 +1,72 @@
+package org.bouncycastle.tls;
+
+import org.bouncycastle.util.io.Streams;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+
+public class TokenBindingExtension {
+
+    public static final Integer rsa2048_pcks15 = 0;
+    public static final Integer rsa2048_pss = 1;
+    public static final Integer rsa2048_ecdsap256 = 2;
```

Comment:
Should these be in upper case since these are constants? Do contrast with the token binding spec as well.

Reply:
spec always denote them in the simple letters